packages/tools/devtools/devtools-core/src/data-visualization/DataVisualization.ts

```diff
@@ -532,24 +532,21 @@ function isSharedObject(value: unknown): value is ISharedObject {
  * {@link @fluidframework/aqueduct#}  is unlikely between devtools and end-user applications, and we don't support it anyway.
  */
 function isDataObject(value: unknown): value is DataObject {
-	if (value instanceof DataObject) {
-		return true;
-	}
-
 	if (
-		(value as DataObject).initializeInternal === undefined ||
-		(
-			value as DataObject as unknown as { getUninitializedErrorString(): string }
-		).getUninitializedErrorString() === undefined
+		value instanceof DataObject ||
+		(typeof (value as DataObject).initializeInternal === "function" &&
+			typeof (
+				value as { getUninitializedErrorString(): string }
+			).getUninitializedErrorString()) === "function"
```

Comment:
Otherwise we're trying to call the function before we know if it exists or is in fact a function.
```suggestion
			).getUninitializedErrorString) === "function"
```